src/shared/components/test/Buttons-test.js

```diff
@@ -0,0 +1,343 @@
+import { mount } from 'enzyme';
+
+import { IconButton, LabeledButton, LinkButton } from '../Buttons.js';
+import { $imports } from '../Buttons.js';
+
+//import { checkAccessibility } from '../../../test-util/accessibility';
```

Comment:
nb. Commented out code here. Did you plan to address that in this PR?

Reply:
Good catch. Yes.

Reply:
Added in latest commits.